test/js/ui/map.test.js

```diff
@@ -292,6 +292,53 @@ test('Map', function(t) {
         }
     });
 
+    t.test('#setMinZoom', function(t) {
+        var map = createMap({zoom:0});
+        map.setMinZoom(3.5);
+        t.deepEqual(map.options.minZoom, 3.5);
```

Comment:
`deepEqual` is for testing the equality of arrays and objects. Since you're just testing a number, you can use `t.equal` here